### PR TITLE
Fix #103 No Open Graph Meta Tags Present

### LINKED ISSUE #103: No Open Graph Meta Tags Present
At present no open graph meta tags are present for stack auth website

scrnli_27_06_2024_17-09-50

scrnli_27_06_2024_17-10-39

### PR DESCRIPTION
**Pull request by Riteshyadavanshi**

### CODE REVIEW

**Comment by imran1khan** [comment]
@Riteshyadavanshi  bro, where is that issue, can you please show me, like a screen short etc

File: apps/dashboard/src/app/layout.tsx [hunk 1]
```diff
@@ -18,11 +18,15 @@ import { RouterProvider } from '@/components/router';
 import { CSPostHogProvider, UserIdentity } from './providers';
 
 export const metadata: Metadata = {
+  metadataBase:new URL("https://stack-auth.com/"),
```
Comment (fomalhautb) [inline apps/dashboard/src/app/layout.tsx:21]: The base url should use the `DASHBOARD_BASE_URL` env var

Comment (fomalhautb) [reply apps/dashboard/src/app/layout.tsx:21]: also please make sure the formatting is aligned with the other part of the project

File: apps/dashboard/src/app/layout.tsx [hunk 2]
```diff
@@ -18,11 +18,15 @@ import { RouterProvider } from '@/components/router';
...
   title: {
     default: 'Stack Auth Dashboard',
     template: '%s | Stack Auth',
   },
   description: 'Stack Auth is the fastest way to add authentication to your web app.',
+  openGraph:{
```
Comment (fomalhautb) [inline apps/dashboard/src/app/layout.tsx:27]: same here

**Comment by Riteshyadavanshi** [comment]
Hey sir ,I have made the changes just now. you can check it

**Comment by fomalhautb** [comment]
Please do not open new PRs multiple times. Just update the PR.

**Pull request closed without merging**